src/makeRobot.js

```diff
@@ -38,7 +38,88 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: null,
+    wheels: null,
+    version: null,
+    x: 0,
+    y: 0,
+
+    addName(item) {
+      this.name = item;
+    },
+
+    addWheels(item) {
+      this.wheels = item;
+    },
+
+    addVersion(item) {
+      this.version = item;
+    },
+
+    get info() {
+      const n = this.name;
+      const v = this.version;
+      const w = this.wheels;
+
+      // Компилятор ругается, что строка ниже слишком длинная видите ли
+      return `name: ${n}, chip version: ${v}, wheels: ${w}`;
```

Comment:
```suggestion
      // Компилятор ругается, что строка ниже слишком длинная видите ли
      return `name: ${n}` 
        + ` chip version: ${v}`
        + ` wheels: ${w}`;
```
можно делать что-то такое